test/attr/ignore_deprecation_warnings_import.swift

```diff
@@ -38,4 +34,7 @@ extension someType: importedDeprecatedProtocol {
     var val: String {
         return ""
     }
-}
\ No newline at end of file
+}
+
+func testDeprecatedConformance<T: ImprotedNonDeprecatedProtocol>(_ x: T.Type) {}
+testDeprecatedConformance(someType.self) // expected-warning {{conformance of 'someType' to 'ImprotedNonDeprecatedProtocol' is deprecated}}
```

Comment:
Shouldn't this be `importedDeprecatedType`? And I believe it shouldn't warn because of the attribute on the import, right?